package-lock.json

```diff
@@ -1,8 +1,921 @@
 {
   "name": "ethereum-multicall",
-  "version": "2.5.0",
-  "lockfileVersion": 1,
+  "version": "2.5.1",
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "version": "2.5.1",
+      "license": "ISC",
+      "dependencies": {
+        "@ethersproject/providers": "^5.0.10",
+        "ethers": "^5.0.15"
+      },
```

Comment:
Please undo this.. your change should only be the files itself. I will update these on the next deployment